libs/designsystem/src/lib/components/checkbox/checkbox.component.spec.ts

```diff
@@ -101,5 +82,37 @@ describe('CheckboxComponent', () => {
 
       expect((spectator.query(IonCheckbox) as IonCheckbox).checked).toBe(false);
     });
+
+    it('should set the [disabled] input on ion-checkbox to true', () => {
+      const disabled = true;
+
+      spectator.setInput('disabled', disabled);
+
+      expect((spectator.query(IonCheckbox) as IonCheckbox).disabled).toBe(true);
+    });
+
+    it('should set the [disabled] input on ion-checkbox to false', () => {
+      const disabled = false;
+
+      spectator.setInput('disabled', disabled);
+
+      expect((spectator.query(IonCheckbox) as IonCheckbox).disabled).toBe(false);
+    });
+
+    it('should have error class when [error] input is true', () => {
+      const error = true;
+
+      spectator.setInput('error', error);
+
+      expect(element).toHaveClass('error');
+    });
+
+    it('should not have error class when [error] input is false', () => {
+      const error = false;
+
+      spectator.setInput('error', error);
+
+      expect(element).not.toHaveClass('error');
+    });
```

Comment:
Consider asserting on the actual color rendered instead using `expect(element).toHaveComputedStyle({...})` (see `libs\designsystem\src\lib\components\button\button.component.spec.ts` for an example)

Reply:
I cant seem to query the actual element inside shadow dom that gets the border.